main/main.cpp

```diff
@@ -2537,10 +2537,11 @@ bool Main::iteration() {
 	if (frame > 1000000) {
 		if (editor || project_manager) {
 			if (print_fps) {
-				print_line("Editor FPS: " + itos(frames));
+				print_line(vformat("Editor FPS: %d (%s mspf)", frames, rtos(1000.0 / frames).pad_decimals(1)));
 			}
 		} else if (GLOBAL_GET("debug/settings/stdout/print_fps") || print_fps) {
-			print_line("Game FPS: " + itos(frames));
+			// Print with leading indentation to make intertwined editor/project output more readable.
```

Comment:
I don't see any leading indentation as referenced in the comment?

Reply:
I left an outdated comment there. Fixed :slightly_smiling_face: